src/component-library/theme/theme.base.css

```diff
@@ -51,6 +51,10 @@
 
   --colors-shared-red: #cb5458;
 
+  --colors-error: #ef4444;
+  --colors-warning: #ff9900;
+  --colors-success: #a2e75e;
+
```

Comment:
Did the Jay provide these colors? I think we may face an issue here as it's not easily possible to use the same colors for both themes because there would be low contrast in one of the variants - notice the green text in right upper corner on interlay:
![image](https://user-images.githubusercontent.com/47864599/182645876-c2f28440-3e61-435e-a0b3-e53c0f913b09.png)
Also we are already using the error, warning and success colors to display transaction statuses. Those colors are different and defined for kintsugi and interlay separately. I suggest we should use the same set of colors in both component lib and older components to keep the dApp consistent. However, these colors are not defined for the component library theme yet, they are defined for the tailwind only (https://github.com/interlay/interbtc-ui/blob/41b454c136998b4ef6f11d0c44c7cdc431e223d0/src/utils/helpers/colors.ts)

I don't think we should block this PR by that, just something to keep in mind that might be useful to bring up when Jay's back.
cc @tomjeatt @crypto-engineer

Reply:
Yes totally agree with this - I would expect there to be defined traffic light colours (per theme) for statuses, which are used consistently. No need to block merging, but could you raise a ticket and assign it to Jay for sprint 43? Don't want it get missed.

Reply:
Yes, I'll create a ticket :+1: